msi/GitProduct.wxs

```diff
@@ -69,7 +69,7 @@
       <!-- Execute the post-install.bat at the end of the installation
            transaction on install and repair but not removal. -->
       <Custom Action="ExecPostInstallBat" Before="InstallFinalize">
-        REMOVE &lt;&gt; "ALL"
+        <![CDATA[REMOVE <> "ALL"]]>
```

Comment:
Sorry, I should have been clearer. After reading the commit message, it is still not obvious to me what this hunk is supposed to change/fix/improve. Is this even needed?